src/Http/Http.Extensions/gen/RequestDelegateGenerator.cs

```diff
@@ -198,6 +200,16 @@ public void Initialize(IncrementalGeneratorInitializationContext context)
                     codeWriter.WriteLine(RequestDelegateGeneratorSources.TryParseExplicitMethod);
                 }
 
+                if (hasEndpointMetadataProvider)
+                {
+                    codeWriter.WriteLine(RequestDelegateGeneratorSources.PopulateEndpointMetadataMethod);
+                }
+
+                //if (hasEndpointParameterMetadataProvider)
```

Comment:
Remove?

Reply:
Yep ... I'm going to add parameter support in a second PR. Just trying to deliver in bite size chunks.